src/main/java/pl/javastart/task/Main.java

```diff
@@ -1,8 +1,52 @@
 package pl.javastart.task;
 
+import java.util.Scanner;
+
 public class Main {
 
     public static void main(String[] args) {
+        final int cellSize = 7;
```

Comment:
stałe deklarujemy jako `static final int CELL_SIZE = 7` na poziomie klasy, nie metody

Reply:
O widzisz. Ja nawet chciałem, ale ciągle dostawałem komunikat ze nie mogę w tym miejscu ustawić stałej. a to po prostu w złym miejscu wstawiałem. dzięki :)